utils/debug/Debug.js

```diff
@@ -133,37 +145,101 @@ function Debug(view, viewerDiv) {
             }
         }
 
+        if (displayed) {
+            nbVisibleChart.update();
+        }
+    }
+}
+
+function ThreeStats(chartDiv, width, left) {
+    createChartContainer(chartDiv, width, left, 'three-info');
 
+    const timestamp = Date.now();
+    const textureDataset = { label: 'texture count', data: [{ x: 0, y: 0 }] };
```

Comment:
is texture count or occupation of the memory (RAM or VRAM), if it's memory what's the unit